### DISCUSSION TITLE
Gradient Designer - For Tailwind 2 - Updated

### DISCUSSION THREAD

**Discussion by jenstornell** [tailwindlabs/tailwindcss#2210] [Show and tell]
### INFO ABOUT THE OLD VERION - SCROLL DOWN FOR NEW VERSION

@adamwathan 

I've built a gradient generator tool, for the new gradient feature in Tailwind 1.7.

https://tailwind-gradient-designer.csspost.com/

What do you think?

![tailwind-gradient-designer csspost com_](https://user-images.githubusercontent.com/927011/90819226-fe70e100-e32f-11ea-9d4d-6141d3cb84ea.png)

**Comment by lucascaton** [comment]
Looks amazing! 👏

**Comment by simonswiss** [comment]
Whoaa this is looking very nice! 🎉

**Comment by jenstornell** [comment]
# Gradient Designer - Minefield 2.0.

It's a complete rewrite in Vue which, uses the screen space better and includes some new features. Also, it includes the new palette from Tailwind 2.

It works fine for me, but I want someone else to try it, before I remove the minefield flag. Please let me know what you think.

**Repo:** https://github.com/jenstornell/gradient-designer
**Tool:** https://gradient-designer.csspost.com/

![screenshot](https://user-images.githubusercontent.com/927011/102121241-f2131580-3e43-11eb-9081-65fb761f3799.png)

**Reply by simonswiss** [reply]
Great stuff man, this is really cool! 🎉
